assets/sass/components/activation/_googlesitekit-activation.scss

```diff
@@ -31,3 +31,19 @@
 		margin-top: 8px;
 	}
 }
+
+.googlesitekit-noscript {
+
+	#googlesitekit-notice-activated {
+
+		border-left-color: $c-selective-yellow; // noscript renders inside success notification parent.
```

Comment:
Sorry, what does this comment mean? I think it's meaning that the override exists because the content is rendered inside a particular component, but the existing comment here isn't totally clear.